src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -540,6 +540,9 @@ public static class RemoveLLOQMode
 
       public static class ContainerName
       {
+         //only use for conversion of older snapshot. Do not use in code otherwise
+         public static readonly string Applications = "Applications";
```

Comment:
can we move this outside and only where this is used then. Does nto make sense to me to define it in a place where it can be used globally